gateway/1.5/modules/ROOT/pages/policies-included-route-weight.adoc

```diff
@@ -0,0 +1,142 @@
+= Traffic Management Policy
```

Comment:
Could you rename this filename to something that matches the policy title? This would be consistent with all the other policy filenames, and it makes it easier to find the policy file.

Something like?
`policies-included-traffic-management.adoc`